CI/travis.osx.install.sh

```diff
@@ -6,13 +6,8 @@ fi
 
 set +e
 shopt -s expand_aliases
-#Removed boost as first item as a temporary workaround to prevent trying to
-#upgrade to boost version 1.68.0 which does not seem to have been bottled yet...
-BREWS="cmake hunspell libzip libzzip lua51 pcre pkg-config qt5 yajl ccache pugixml luarocks"
+BREWS="boost cmake hunspell libzip libzzip lua51 pcre pkg-config qt5 yajl ccache pugixml luarocks"
 for i in $BREWS; do
-  if [ "${i}" = "cmake" ]; then
```

Comment:
Yeah, this is a non-stopper (was going to say non-*starter* but that is not really how things are) because `pugixml` pulls in `cmake` anyhow...

Reply:
But only when building pugixml from source. When installing the bottles version (which only exists for Sierra and up now) then cmake is no dependency.